### ISSUE TITLE
Polaris Webhook doesn't work with Kubernetes v1.16

### ISSUE THREAD

**Issue by joschi36** [FairwindsOps/polaris#228]
### Installation Process
Helm

### Polaris Version
```
Polaris: 0.5.2
Kubernetes: 1.16.3
Helm: 3.0.0
```

### Expected Behavior
Webhook Server Starts

### Actual Behavior
```
time="2019-12-06T09:12:24Z" level=info msg="Setting up webhook server"
time="2019-12-06T09:12:24Z" level=info msg="Polaris webhook server listening on port 9876"
time="2019-12-06T09:12:24Z" level=info msg="deployments-0-0.k8s.io webhook started"
time="2019-12-06T09:12:24Z" level=error msg="Error building webhook: no matches for kind \"Deployment\" in version \"extensions/v1beta1\""
```

### Steps to Reproduce
```
$ helm upgrade --install polaris reactiveops-stable/polaris --namespace polaris --set webhook.enable=true --set dashboard.enable=false
$ kubectl logs polaris-webhook-8474cdbfbd-zs6bv
```

### Additional Context
`extensions/v1beta1` was removed in v1.16 (https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/)

**Comment by rbren** [comment]
Thanks for the report!

We've got a potential fix in https://github.com/FairwindsOps/polaris/pull/229.  Can you try another `helm upgrade` with `--set webhook.image.tag=rb_k8s-116-support` to verify?

**Comment by joschi36** [comment]
thanks @rbren #229 seems to work :+1:
Looking forward to a release ;)

**Comment by rbren** [comment]
Released in 0.6.0. Thanks again!
